CurrencyControl/CurrencyControl.cs

```diff
@@ -0,0 +1,199 @@
+﻿using CurrencyControl.Data;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Controls.Primitives;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace CurrencyControl
+{
+    [TemplatePart(Name = "BtnConvert", Type = typeof(Button))]
+    public class CurrencyControl : Control
```

Comment:
All public classes and their public members should be commented.

Please check for more details:
[Coding Style - ComponentOne - Confluence (atlassian.net)] https://grapecity.atlassian.net/wiki/spaces/C1/pages/2933164273/Coding+Style)
[Recommended XML tags for C# documentation comments](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags)

Reply:
Changes pushed.